system-tests/src/test/java/org/eclipse/dataspaceconnector/system/tests/identityhub/IdentityHubIntegrationTest.java

```diff
@@ -38,7 +41,7 @@
 import static org.eclipse.dataspaceconnector.system.tests.utils.TestUtils.requiredPropOrEnv;
 import static org.junit.jupiter.params.provider.Arguments.arguments;
 
-public class IdentityHubIntegrationTest {
+class IdentityHubIntegrationTest {
 
     static final String PROVIDER_IDENTITY_HUB_URL = requiredPropOrEnv("PROVIDER_IDENTITY_HUB_URL", "http://localhost:8181/api/identity-hub");
```

Comment:
I know its unrelated to this PR, but I think it would be good to rename these constants according to the naming of the participants:
- PROVIDER_IDENTITY_HUB_URL -> COMPANY1_IDENTITY_HUB_URL
- CONSUMER_EU_IDENTITY_HUB_URL  -> COMPANY2_IDENTITY_HUB_URL
- CONSUMER_US_IDENTITY_HUB_URL  -> COMPANY3_IDENTITY_HUB_URL

Reply:
Fine for you if I take this in the next PR when I will add the self-description for the authority? As I was planning to do some updates here.

Reply:
Yes, we can do this cleanup later